packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -536,16 +536,16 @@ class FuchsiaDevice extends Device {
   @override
   Future<void> takeScreenshot(File outputFile) async {
     if (outputFile.basename.split('.').last != 'ppm') {
-      throw '${outputFile.path} must be a .ppm file';
+      throw Exception('${outputFile.path} must be a .ppm file');
```

Comment:
If the call site of `takeScreenshot` is catching a `String` then it would need to be updated. What do the `takeScreenshot` implementations on other devices do?

Reply:
The call site is actually catching an `Exception` (https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/screenshot.dart#L132), and I don't think this is using a String to bypass that catch block.

Other platforms don't have as much code in `takeScreenshot`, they either throw `UnimplementedError` or call `ProcessManager` to actually take screenshots